libkbfs/config_local.go

```diff
@@ -97,7 +97,7 @@ type ConfigLocal struct {
 	kbpki            KBPKI
 	renamer          ConflictRenamer
 	registry         metrics.Registry
-	loggerFn         func(prefix string) logger.Logger
+	loggerFn         func(prefix string, overrideEnableDebug bool) logger.Logger
```

Comment:
Why is there an "enable" in the param name?  Couldn't it just be `overrideDebug`?  Here are everywhere else.

Reply:
Because setting it to false doesn't disable debug if it was already enabled through `params` [here](https://github.com/keybase/kbfs/pull/1375/files#diff-2bd2766c2ea1726e9c09c18beb70ee12R576). It could be a `overrideDebug *bool` though, which if non-nil overrides `params.Debug`. WDYT?

Reply:
I think we have different definitions for the word "override" or something.  If a boolean called "overrideDebug" is false, I would assume that it doesn't change anything about the debug value, it just leaves it alone.  That's what this does, right?

Reply:
Oh I see; that makes sense. Will update the word then. Thanks!

Reply:
so make it `ensureDebugEnabled`

Reply:
@jzila Thanks; `overrideDebug` actually sounds good to me now; but let me know if it bothers you and I'll change it into `ensureDebugEnabled`